### ISSUE TITLE
tootctl accounts create --approve

### ISSUE THREAD

**Issue by kg4zow** [mastodon/mastodon#22870]
### Pitch

After using `tootctl accounts create` to create a user, the admin then has to go into the web interface and "approve" the new user, or they have to use `tootctl accounts modify` with the `--approve` option to do the same thing.

I can't imagine it would be too difficult to add an `--approve` option to the `tootctl accounts create` command.

### Motivation

This would streamline the process of creating users from the command line.

**Comment by ineffyble** [comment]
From looking at the code, it seems like any account created with `tootctl accounts create` should be automatically approved.

is that not the case?

**Comment by kg4zow** [comment]
That's not what happened for me.

I had created a second user with the "Owner" role, but when I logged into it from a different browser, it told me that the account needed to be "approved" before I could do anything. Back in the first browser as the `admin` user I figured out how to "approve" a user through the web interface, then the new user was able to access everything.

I added a third user (also with the "Owner" role) and had the same results, and used this as an excuse to figure out how to "approve" a user through the command line.

Running v4.0.2, installed on a fresh Debian 11 VM using the directions from the web site.
